egui/src/widgets/plot.rs

```diff
@@ -35,7 +36,7 @@ impl Value {
 
 /// 2D bounding box of f64 precision.
 /// The range of data values we show.
-#[derive(Clone, Copy, PartialEq)]
+#[derive(Clone, Copy, PartialEq, Serialize, Deserialize, Debug)]
```

Comment:
It's strange that CI does not complain about this line. `persistence` feature is not enabled by default and I get errors when `cargo check` `egui` folder locally. @emilk CI didn't check this.

You should use serde like this:
```suggestion
#[derive(Clone, Copy, PartialEq, Debug)]
#[cfg_attr(feature = "persistence", derive(serde::Deserialize, serde::Serialize))]
```

Reply:
Thanks for the review!

`cargo check` in the `egui` folder works for me even without the changes...